### PR TITLE
add new article

### PR DESCRIPTION
**Pull request by freeman-lab**

### CODE REVIEW

File: articles/cdr-scale-barriers/components/big-quote.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            bg: 'background',
+            pl: [4],
+            color: COLORS[category],
+          }}
+        >
+          {category}
+        </Box>
+      </Box>
+      <Row columns={6}>
+        <Column start={1} width={5}>
```
Comment (katamartin) [inline articles/cdr-scale-barriers/components/big-quote.js:45]: At the largest breakpoint, the quote content feels a _little_ wide. We could try narrowing to four columns at that breakpoint. Don't feel particularly strongly about this though!

now:
Screen Shot 2022-07-12 at 8 32 47 AM
alternative:
Screen Shot 2022-07-12 at 8 32 32 AM

File: articles/cdr-scale-barriers/components/download-report.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              className='arrow'
+              sx={{ ...sx, ml: [0], color: 'teal' }}
+            />
+            <RotatingArrow className='arrow' sx={{ ...sx, color: 'green' }} />
+            <RotatingArrow className='arrow' sx={{ ...sx, color: 'orange' }} />
+            <RotatingArrow className='arrow' sx={{ ...sx, color: 'yellow' }} />
+          </>
+        }
+        sx={{
+          '&:hover > #suffix-span > .arrow': {
```
Comment (katamartin) [inline articles/cdr-scale-barriers/components/download-report.js:38]: nit: It seems a little brittle to reference the internal ids defined in `Button`. If we put a class on a span wrapping the `RotatingArrows` we could do `&:hover .arrow-wrapper > .arrow`

Comment (freeman-lab) [reply articles/cdr-scale-barriers/components/download-report.js:38]: great idea, fixed!

**Review by katamartin — APPROVED** [review]
looks great!

**Pull request merged**